moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -150,6 +150,12 @@ struct CollisionRequest
    * are included. */
   std::string group_name = "";
 
+  /** \brief If true, use padded collision environment */
+  bool use_padded_collision_environment = true;
```

Comment:
Should these names be more consistent with each other?

Like `pad_environment_collisions` and `pad_self_collisions`?

Or `use_padded_environment_collisions` and `use_padded_self_collisions`?

Reply:
Also, is it intentional for the defaults to be different? I assume the defaults maintain the old behavior?

Reply:
Yes, exactly that's why I made them different. At the moment, we're using by default padding for environment collision checks and no padding for self-collision checks